app/models/user.rb

```diff
@@ -2,10 +2,10 @@ class User < ApplicationRecord
   validates :email, uniqueness: true, presence: true
   validates :password, presence: true
 
-  has_many :followed_users, foreign_key: :follower_id, class_name: 'Friend', dependent: :destroy
+  has_many :followed_users, foreign_key: :follower_id, class_name: 'Friend', dependent: :destroy, inverse_of: :follower
```

Comment:
This makes sense but I wonder why rubocop asks us to add it since it works without. Something to ask the instructor